index.html

```diff
@@ -2296,6 +2296,41 @@ <h1>Core Representations</h1>
 representations should be added here once that section has been written.
     </p>
 
+    <section>
+      <h2>Handling Properties</h2>
+      
+        <section>
+          <h2>Unknown Properties</h2>
+          <p>
+    When a DID Document is created by consuming a compliant representation, any properties
+    in the representation that are not known to the consumer MUST be added to the
+    data model as properties without any further processing on the property
+    value.
+          </p>
+          <p>
+    When a DID Document is serialized by a producer into a compliant
+    representation, any unknown properties MUST be serialized into the
+    representation. When the data type for the property is unknown by the producer,
+    a string serialization of the property SHOULD be used.
+          </p>
+        </section>
+  
+        <section>
+          <h2>Reserved Properties</h2>
+          <p>
+    When a DID Document is created by consuming a compliant representation, if any 
+    properties in the representation are reserved by another representation, as 
+    indicated in the registry entry for representations, then the consumer MUST
+    raise an error. 
```

Comment:
I take it this statement means that 2 representations cannot both contain the same reserved property.... so if JSON-LD reserved `$schema` it could not be present in JSON Documents or vice versa?

I suppose this will cut down on representations and reservations :)

Reply:
That's not the intent, as a representation could declare that it uses another representation's reserved property in a way that's compatible with that. The important thing about a reserved property is that it doesn't end up in the abstract data model.

Reply:
Ahh, `deny-list` and `allow-list` inheritance... the road to a super confusing data model...

But I do agree that there will be representations that want to specifically support things allowed and prohibited by other representations... the first one would be `application/did+vanilla+json`... which allowed directives like `@context` and `$schema`.

Reply:
> The important thing about a reserved property is that it doesn't end up in the abstract data model.

What concrete attack or issue are we attempting to prevent here?

Reply:
No inheritance. Concrete declaration of using the same property to mean the same thing. So JSON-LD and CBOR-LD could both claim "@context" as reserved, which signals everyone else that if they see that field and don't know what to do with it (ie, it's not reserved for them also) then it's an error.

Reply:
JSON does not throw errors when random properties are added.... what you are describing is not JSON.